diff --git a/CMakeLists.txt b/CMakeLists.txt
index d5274b7..8798efe 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -8,7 +8,6 @@ set(CMAKE_CXX_STANDARD_REQUIRED YES)
 # Find catkin macros and libraries
 find_package(catkin REQUIRED COMPONENTS
   roscpp
-  roslint
 )
 
 # System dependencies are found with CMake's conventions
@@ -95,5 +94,3 @@ if(CATKIN_ENABLE_TESTING)
   find_package(rostest REQUIRED)
 endif()
 
-## Test for correct C++ source code
-roslint_cpp()
